Rakefile

```diff
@@ -113,6 +119,25 @@ rescue
   puts 'Rubocop not loaded'
 end
 
+namespace :config do
```

Comment:
I think the Rails use case and the Kafo use case are much different, it's very difficult for me to look through a dozen migrations + the answer file to figure out what our canonical value is for a new install